dune-project

```diff
@@ -2,3 +2,27 @@
 (name lambdapi)
 
 (using menhir 1.0)
+
+(generate_opam_files true)
+(source (github Deducteam/lambdapi))
+(license CeCILL2.1)
+(maintainers "dedukti-dev@inria.fr")
+(package
+ (name lambdapi)
+ (synopsis "Proof assistant based on the λΠ-calculus modulo rewriting")
+ (description "Proof assistant based on the λΠ-calculus modulo rewriting,
+mostly compatible with Dedukti.
+
+This package contains both the `lambdapi` proof assistant and the `lp-lsp`
+Language Server Protocol helper for IDEs to check lambdapi files.")
+ (depends
+  (ocaml (>= 4.07.0))
+  (menhir (>= 20200624))
+  (alcotest :with-test)
+  (bindlib (>= 5.0.1))
+  (timed (>= 1.0))
+  (pratter (>= 1.1))
+  (why3 (>= 1.3.1))
```

Comment:
We currently need to restrict why3 to <= 1.3.3. See https://github.com/Deducteam/lambdapi/pull/608.